controllers/productController.js

```diff
@@ -1,44 +1,39 @@
-const Product = require("../models/productModel");
-const serviceProduct = require("../models/productModel");
+const serviceProduct = require('../services/productService');
```

Comment:
lets use destruction instead of access function using serviceProduct . const {addProduct}=require('../services/productService')